packages/commons-test/src/mockedPayloadForToken.ts

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 import { AuthRole, userRole } from "pagopa-interop-commons";
-import { UserId, generateId } from "pagopa-interop-models";
+import { TenantId, UserId, generateId } from "pagopa-interop-models";
 import { match } from "ts-pattern";
 import jwt from "jsonwebtoken";
-import { getMockAuthData } from "./testUtils.js";
 
-function createUserPayload(role: AuthRole) {
+export const mockTokenOrganizationId = generateId<TenantId>();
+
+function createUserPayload(commaSeparatedUserRoles: string) {
```

Comment:
Changes to this method are a cherry pick from https://github.com/pagopa/interop-be-monorepo/pull/1863 to minimize conflicts